src/dev-app/theme-token-api.scss

```diff
@@ -16,33 +18,47 @@ dev-app {
   }
 }
 
-.demo-unicorn-dark-theme {
-  background: black;
-  color: white;
-}
-
+// Emit Angular Material core styles.
 @include mat.core();
 
-$theme: mat.define-light-theme((
+// Base theme configuration for our M3 theme.
+$m3-base-config: (
   color: (
-    primary: mat.define-palette(mat.$indigo-palette),
-    accent: mat.define-palette(mat.$pink-palette),
+    primary: matx.$m3-green-palette
   ),
-  typography: mat.define-typography-config(),
-  density: 0,
-));
+  typography: (
+    brand-family: 'monospace'
```

Comment:
Where does the terminology come from? Does it align with MDC or is it something we came up with?

Reply:
This is from the Material Design spec: https://m3.material.io/styles/typography/type-scale-tokens#48e80519-e9c3-474b-9eb0-4adaf2850595